pyproject.toml

```diff
@@ -53,6 +53,10 @@ dev = [
     "tox",
 ]
 allenmouse = ["allensdk"]
+allenmouse_barrels = [
+    "allensdk",
+    "atlas-enhancement"]
```

Comment:
I have asked the reason for a specific numpy version, awaiting response.
Indeed I am not sure one needs to be strict about this.

Reply:
This would be the quick instructions to install atlas-enhancement and run the script:

1. Clone the fork of atlas-enhancement repository (https://github.com/abisi/atlas-enhancement) into BrainGlobe's working directory
2. Download annotation files in `mouse_enhanced_cortex.zip` from Zenodo: https://zenodo.org/records/11218079
3. Move the 'hierarchy.json' and 'annotation_barrels.nrrd' files into a new `data` folder in `atlas-enhancement\barrel-annotations\data`.
4. Run the script `allen_mouse_bluebrain_barrels.py`